app/views/external_users/json_document_importers/create.js.erb

```diff
@@ -73,7 +68,7 @@ $importerResults.find('.js-import-failed').remove();
       '</ul>' +
     '</div>'
   );
-  $('#importer').addClass('has-errors');
+  $('#importer').addClass('has-errors').find('form').addClass('no-file-selected').find('.file-upload-name').empty();
```

Comment:
It looks like this form field reset function should be extracted to a function, as it's repeated three times in this change alone.